heapster-base/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Heapster Docker Base Image
+FROM centos:centos7
+
+# The image is maintained by the Hawkular Metrics team
+MAINTAINER Hawkular Metrics <hawkular-dev@lists.jboss.org>
+
+EXPOSE 8082
+
+ENV HEAPSTER_COMMIT=af4752e
+
+RUN yum install -y go git wget make && \
+    yum clean all
+
+RUN export GOPATH=/tmp/gopath && \
```

Comment:
We can. I usually like to keep things a bit separate if I think it makes sense logically (eg 1) setup system; 2) build heapster). I usually put a higher value of readability and separation of concerns versus having an extra layer.